salesforce/10.10/modules/ROOT/pages/salesforce-connector-studio.adoc

```diff
@@ -72,367 +74,242 @@ Subscribe to a topic. This source provides topic-related notification settings f
-
-This procedure assumes that you already have a certification file (such as `salesforce-cert.crt`). If not, you can produce one by generating a Java keystore and public key.
-
-[[create-consumer-key]]
-. Log in to Salesforce.
-. From Setup, enter `Apps` in the *Quick Find* box.
-. Click *App Manager*.
-. Click *New Connected App*.
-. To create a new connected app, enter:
-+
-* A name for the connected app
-* The API name
-* The contact email
-+
-. In the *API (Enable OAuth Settings)* section, select the *Enable OAuth Settings* checkbox:
-+
-* Enter the *Callback URL*.
-* Select the *Use Digital Signatures* checkbox.
-* Click *Choose File* and load your Salesforce certificate (for example, `salesforce-cert.crt`), which contains your public key.
-+
-In Studio, you typically store the certificate in the workspace that contains your Mule app.
-+
-. Add and save these OAuth scopes to *Selected OAuth Scopes*:
-+
-** *Full Access* (`full`)
-** *Perform Requests On Your Behalf At Any Time* (`refresh_token`, `offline_access`)
-+
-. Configure the authorization settings for the app.
-. Click *Manage*.
-. In the OAuth Policies section, expand the *Permitted Users* list and select *Admin Approved Users are Pre-Authorized*.
-+
-. Click *Save*.
-. In the Profiles section, click *Manage Profiles*.
-. Select your user profile and click *Save*.
-. Select *Build* > *Create* > *Apps* to return to the list of connected apps.
-. In the Connected Apps section, select the connected app you created.
-
-You can now see the consumer key that you need to provide in your connector's configuration.
-
-== Generate a Keystore File
-
-This example demonstrates how to create a JKS keystore (PKCS12 format is also supported).
-
-The Key Store field is the path to the keystore used to sign data during authentication.
-
-[NOTE]
-Salesforce Connector is using the Bouncy Castle cryptographic library to load the certificate from the keystore and sign the payload for the authentication requests.
-The Bouncy Castle library has a https://nvd.nist.gov/vuln/detail/CVE-2018-5382[vulnerability (CVE-2018-5382)] that is related to the BKS-V1 keystore file type.
-By default, Salesforce Connector is using JKS or PKCS12 keystores. It is best not to use BKS-V1 keystore files, as documented in the https://www.kb.cert.org/vuls/id/306792[Bouncy Castle proposed solution].
-
-To generate a keystore file:
-
-. Go to your Mule workspace and open the command prompt (for Windows) or Terminal (for Mac).
-. Type this command and press enter:
-+
-[source]
-----
-keytool -genkeypair -alias salesforce-cert -keyalg RSA -keystore salesforce-cert.jks
-----
-+
-. Enter the following information:
+* *Type*
 +
-** Password for the keystore
-** Your first name and last name
-** Your organization unit
-** Name of your city, state, and the two-letter code for your country
+Salesforce object type
```

Comment:
Note that per adjusted bullet list entry in style ref, you can now use this format instead: your choice.

* *Type*: Salesforce object type
* *Records*: Function editor expression....

Reply:
Good to know for the future